PowerEditor/installer/nativeLang/french.xml

```diff
@@ -270,11 +270,11 @@
                     <Item id="45001" name="Convertir en format Windows (CR+LF)"/>
                     <Item id="45002" name="Convertir en format UNIX (LF)"/>
                     <Item id="45003" name="Convertir en format Mac (CR)"/>
-                    <Item id="45004" name="Encoder en ANSI"/>
-                    <Item id="45005" name="Encoder en UTF-8 (avec BOM)"/>
-                    <Item id="45006" name="Encoder en UTF-16BE"/>
-                    <Item id="45007" name="Encoder en UTF-16LE"/>
-                    <Item id="45008" name="Encoder en UTF-8"/>
+                    <Item id="45004" name="Fichier encodé en ANSI"/>
```

Comment:
I don't see any advantage in your modification, furthermore you make the active action to a passive state.

Reply:
Merci pour ta réponse. Puisqu'on commente du texte français, je continue dans cette langue.
J'ai toujours eu du mal à distinguer les entrées "Encoder en UTF-8" et "Convertir en UTF-8" par exemple. Et au final, voici ce que j'ai compris du fonctionnement : 

A l'ouverture d'un fichier texte, N++ essaie de détecter l'encodage du fichier. Supposons qu'il identifie ANSI : l'entrée "Encoder en ANSI" est alors précochée et "ANSI" est affiché dans la barre d'état.  Si je veux encoder mon fichier en UTF-8, je dois choisir l'entrée de menu "Convertir en UTF-8".
Les entrées de menu "Encoder en UTF-8" & co  me semble être le moyen pour l'utilisateur d'indiquer l'encodage du fichier, à utiliser au cas où N++ ne serait pas parvenu à identifier correctement l'encodage du fichier.. D'où ma proposition sous la forme passive : j'indique à N++ que le fichier est encodé en UTF-8 par exemple. Comme N++ m'indique à l'ouverture que le fichier est encodé en ANSI.
Est-ce que je me trompe ?

En tout cas, je te remercie pour ton travail. N++ est utilisé partout où j'ai pu travailler. C'est un très bon produit.
Cordialement

Reply:
Agree with SunWukong, I also got confused with the current wording several times.

I'd suggest "Encoded" in English and just "Encodé" in French.

Reply:
As  Don explained, “you make the active action to a passive state” — I think that would confuse much more users.

Reply:
Hi, 
Je suis d'accord avec @sunWukong et @vlakoff, j'opterais pour "Encodé en xx".

>Les entrées de menu "Encoder en UTF-8" & co me semble être le moyen pour l'utilisateur d'indiquer l'encodage du fichier, à utiliser au cas où N++ ne serait pas parvenu à identifier correctement l'encodage du fichier.

Exact,
pour l'option "encodé en ..." on force notepad++ à recharger le fichier en lui demandant de l'interprété avec l'encodage sélectionné, l'affichage du fichier est différent mais son contenue sur le disque est identique.
>The option Encode in .... OR Character sets / .... / ...., apply the chosen encoding to the actual contents of the current file. The contents of the current file, after encoding, are not modified and ONLY displaying is changed.

Pour l'option "convertir en ..." le fichier est modifier sur le disque dur et l'affichage reste le même.
>The option Convert to... transforms the current file, with its actual encoding, to the same contents file, translated in the chosen encoding.

Veuillez également noter les 2 points suivants :
- lorsque notepad++ identifie le codage d'un fichier comme UTF-8 **avec BOM** ou UCS-2 LE **avec BOM**, la fonction "encodé en ANSI" commet une erreur car seul la partie **après la BOM** est rechargé (il manque donc les 2 ou 3 octets constituant la BOM).
- lorsque notepad++ identifie le codage UCS2-xx (codage d'1 caractère sur 1 ou 2 octets) la fonction "encodé en ANSI" recharge le fichier en oubliant des octets ce qui donne quelque chose complètement différent du contenue du fichier sur le disque.

Ces 2 derniers points sont expliqués et corrigés dans #3764 et #3836

Dans le menu il manque également des options pour les encodage : UCS2-xx sans BOM voir #5899 et #5900

Reply:
Furthermore, there's a mark (a kind of bullet) on the left of the menu entry "Encoder en ANSI" which really sounds like "Encodé en ANSI". 
The problem is that it looks like an indication, it actually is an indication and it is also an action to reload the file with the choosen encoding. So both a passive and an active thing. That indication may be redundant with the one displayed in the status bar.

IMHO, as many of us are struggling everyday with encoding problems, with more or less undestanding, it may be good to clarify those entries. Maybe by moving the "reload with choosen encoding" entries in a more advanced OpenFile Dialog ?

Reply:
I see the point.
Indeed, these commands are quite confusing. They're the active commands and state (with the indicator) at the same time.
I would use Interpret as (in French Interprêter en tant que), but these commands do modify the file by adding or removing BOM (3 bytes) at the beginning of file content after saving.

If you have better name, please let me know.

Reply:
2 solutions come in my mind:
```
Set as ANSI
Set as UTF-8
Set as UTF-8 BOM
Set as UCS-2 BE BOM
Set as UCS-2 LE BOM
```
or just:
```
ANSI
UTF-8
UTF-8 BOM
UCS-2 BE BOM
UCS-2 LE BOM
```
Which one you guys prefer?

Reply:
I think people would be confused between "Set as" and "Encode in"...

Reply:
I think people would be confused between "Set as" and "Convert to"...

Reply:
After some thought, your 2 solutions look fine to me.

Reply:
I think this is closer to the action performed when we click on it:

English
```
Reload as ANSI
Reload as UTF-8
Reload as UTF-8 BOM
Reload as UCS-2 BE BOM
Reload as UCS-2 LE BOM
```

French:
```
Recharger en tant que ANSI
Recharger en tant que UTF-8
Recharger en tant que UTF-8 BOM
Recharger en tant que UCS-2 BE BOM
Recharger en tant que UCS-2 LE BOM
```



but the simplest is
````
ANSI
UTF-8
UTF-8 BOM
UCS-2 BE BOM
UCS-2 LE BOM
````

Reply:
I much prefer Cyrillev91's proposal : I think it is much more understandable.
The actual encoding of the file being displayed in the status bar, I keep thinking it is redundant in the menu (as a displayed information). 
For "reloading as" or "convert to", it would be nice to put that choice in the FileOpen() FileSave() dialog boxes, under "Open File" and "Save as". But much work...

Reply:
"Reload" or "Recharger" is for an existing file on disk. So even we could ignore the fact these action can change the file content, it's sttill not accurate because these actions can be applied to the content of a new document.
In these case, I prefer only
````
ANSI
UTF-8
UTF-8 BOM
UCS-2 BE BOM
UCS-2 LE BOM
```
or 
````
As ANSI
As UTF-8
As UTF-8 BOM
As UCS-2 BE BOM
As UCS-2 LE BOM
```

Reply:
I'd bend towards just "ANSI / UTF-8 / etc."

As a bonus, it would be consistent with the status bar.
(btw, a small [reminder](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/5523) :)

Reply:
Ok for
```ANSI
ANSI
UTF-8
UTF-8 BOM
UCS-2 BE BOM
UCS-2 LE BOM
````
Simpler and consistent with : 
- `Encoding >>character Set >> xxxxx>> xxxxx` menu
- status bar

Reply:
Agreed. Thanks